src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.tsx

```diff
@@ -159,6 +165,28 @@ const PowerBIWidgetBuilder: React.FC<IPowerBIWidgetBuilderProps> = ({
         [formData, reportVisuals, updateWidgetData]
     );
 
+    const renderReportUrlLabel = (props: ITextFieldProps) => {
+        return (
+            <Stack
+                horizontal
+                verticalAlign="center"
+                tokens={{
+                    childrenGap: 4
+                }}
+            >
+                <Label id={props.id} htmlFor={reportUrlId}>
```

Comment:
I have never seen `htmlFor` before. Is that like aria-labeled-by but in reverse?

Reply:
It gets the label value of the referenced html tag.

Reply:
Kinda that and also if the user clicks a <label for="myControl" /> the element with id/name "myControl" gets focused.